tests/test_validate.py

```diff
@@ -0,0 +1,49 @@
+"""Test for sorting MappingSetDataFrame columns."""
+
+import unittest
+
+from jsonschema import ValidationError
+
+from sssom.constants import DEFAULT_VALIDATION_TYPES, SchemaValidationType
+from sssom.parsers import parse_sssom_table
+from sssom.validators import validate
+from tests.constants import data_dir
+
+
+class TestValidate(unittest.TestCase):
+    """A test case for sorting msdf columns."""
+
+    def setUp(self) -> None:
+        """Test up the test cases with the third basic example."""
+        self.correct_msdf1 = parse_sssom_table(f"{data_dir}/basic.tsv")
+        self.bad_msdf1 = parse_sssom_table(f"{data_dir}/bad_basic.tsv")
+        self.validation_types = DEFAULT_VALIDATION_TYPES
+        self.shacl_validation_types = [SchemaValidationType.Shacl]
+
+    def test_validate_json(self):
+        """Test JSONSchemaValidation."""
```

Comment:
make a better comment saying why this should fail or pass.

Reply:
Done.